playbooks/hosts/amc_production/group_vars/edx

```diff
@@ -0,0 +1,204 @@
+COMMON_DEPLOYMENT: "default-staging"
+
+## Theme
+edxapp_use_custom_theme: false # false to disable & use std. theme
+edxapp_theme_name: "default"
+edxapp_theme_source_repo: "https://github.com/appsembler/edx-theme.git"
+edxapp_theme_version: "master" #branch
+
+#disable analytics link in instructor dashboard
+#EDXAPP_ANALYTICS_DASHBOARD_URL: ""  # if enabled: http://insights.examples.com
+
+## NGINX Config
+NGINX_ENABLE_SSL: true
+NGINX_REDIRECT_TO_HTTPS: true
+NGINX_SSL_CERTIFICATE: "/Users/bezidejni/Documents/projekti/appsembler/appsembler.crt"
+NGINX_SSL_KEY: "/Users/bezidejni/Documents/projekti/appsembler/appsembler.key"
+NGINX_SET_X_FORWARDED_HEADERS: True #set to True if not behind load balancer
+EDXAPP_LMS_NGINX_PORT: 80
+EDXAPP_LMS_SSL_NGINX_PORT: 443
+EDXAPP_LMS_PREVIEW_NGINX_PORT: 80
+EDXAPP_CMS_NGINX_PORT: 80
+EDXAPP_CMS_SSL_NGINX_PORT: 443
+CMS_HOSTNAME: '~^((stage|prod)-)?studio.*' #for redirect within nginx
+
+nginx_sites:
+  - certs
+  - cms
+  - lms
+  - forum
+  - xqueue
+nginx_default_sites:
+  - lms
+#COMMON_HTPASSWD_USER: "user"
+#COMMON_HTPASSWD_PASS: "password"
+COMMON_ENABLE_BASIC_AUTH: False
+
+NGINX_ROBOT_RULES:
+  - { agent: '*', disallow: '/' }
+
+## Base
+EDXAPP_LMS_BASE: "academy.appsembler.com"
+EDXAPP_PREVIEW_LMS_BASE: "academy-preview.appsembler.com"
+EDXAPP_CMS_BASE: "academy-studio.appsembler.com"
+CMS_HOSTNAME: "academy-studio.appsembler.com"
+
+## Site names
+EDXAPP_SITE_NAME: "academy.appsembler.com"
+EDXAPP_LMS_SITE_NAME: "academy.appsembler.com"
+EDXAPP_CMS_SITE_NAME: "academy-studio.appsembler.com"
+
+## Social media
+EDXAPP_PLATFORM_FACEBOOK_ACCOUNT: "http://www.facebook.com/accountName"
+EDXAPP_PLATFORM_TWITTER_ACCOUNT: "@twitterHandle"
+
+## Email addresses
+EDXAPP_CONTACT_EMAIL: "customerservice@example.com"
+EDXAPP_DEFAULT_FEEDBACK_EMAIL: "customerservice@example.com"
+EDXAPP_DEFAULT_FROM_EMAIL: "customerservice@example.com"
+EDXAPP_BUGS_EMAIL: "bugs@appsembler.com"
+EDXAPP_BULK_EMAIL_DEFAULT_FROM_EMAIL: "customerservice@example.com"
+EDXAPP_PAYMENT_SUPPORT_EMAIL: "customerservice@example.com"
+EDXAPP_PRESS_EMAIL: "customerservice@example.com"
+EDXAPP_SERVER_EMAIL: "devops@appsembler.com"
+EDXAPP_TECH_SUPPORT_EMAIL: "technical@appsembler.com"
+EDXAPP_FEEDBACK_SUBMISSION_EMAIL: "customerservice@example.com"
+EDXAPP_UNIVERSITY_EMAIL: "customerservice@example.com"
+
+## Email backend
+EDXAPP_EMAIL_BACKEND: "django.core.mail.backends.smtp.EmailBackend"
+#EDXAPP_EMAIL_BACKEND: "django_ses.SESBackend"
+#EDXAPP_EMAIL_HOST: "email-smtp.us-..."
+#EDXAPP_EMAIL_PORT: 25
+#EDXAPP_EMAIL_HOST_USER: "xxxxxx"
+#EDXAPP_EMAIL_HOST_PASSWORD: "xxxxx"
+#EDXAPP_EMAIL_USE_TLS: true
+
+# AWS S3 configuration
+#EDXAPP_AWS_ACCESS_KEY_ID: "xxxxx"
+#EDXAPP_AWS_SECRET_ACCESS_KEY: "xxxxx"
+#EDXAPP_AWS_STORAGE_BUCKET_NAME: "edx-NAME-bucket"
+
+#grade downloads
+#EDXAPP_GRADE_BUCKET: "edx-NAME-grade-bucket"
+#EDXAPP_GRADE_STORAGE_TYPE: "S3"
+#EDXAPP_GRADE_ROOT_PATH: "grades"
+
+# Setting up New Relic
```

Comment:
Why are the New Relic lines commented out? We should have New Relic monitoring the Appsembler Academy site.